libs/aws/langchain_aws/chat_models/bedrock.py

```diff
@@ -477,14 +477,19 @@ def _stream(
             **kwargs,
         ):
             if isinstance(chunk, AIMessageChunk):
-                yield ChatGenerationChunk(message=chunk)
+                generation_chunk = ChatGenerationChunk(message=chunk)
+                if run_manager:
```

Comment:
Adding a check to test for empty/None values will fix the 2 empty responses in the end:
```
 if run_manager and generation_chunk.text:
```